baselines/statavg/README.md

```diff
@@ -48,64 +47,60 @@ dataset: [TON_IoT]
 | data partition | stratified based on labels (6 classes per client) |
 | optimizer | Adam |
 
-
 ## Environment Setup
 
 To construct the Python environment, simply run:
 
 ```bash
-# Set directory to use python 3.10 (install with `pyenv install <version>` if you don't have it)
-pyenv local 3.10.13
+# Create the virtual environment
+pyenv virtualenv 3.10.14 statavg
 
-# Tell poetry to use python 3.10
-poetry env use 3.10.13
+# Activate it
+pyenv activate statavg
 
-# Install
-poetry install
+# Install the baseline
+pip install -e .
 ```
 
 ## Dataset Preparation
+
 You can download the TON_IoT dataset by accessing the following [link](https://research.unsw.edu.au/projects/toniot-datasets). Please navigate to `TON_IoT datasets/Train_Test_datasets/Train_Test_Linux_dataset` and download the file `Train_test_linux_memory.csv`. Then, rename the downloaded file to `dataset.csv` and place it in the `dataset/` directory. If you want to run the experiments with your own data, ensure your dataset is also named `dataset.csv` and located in the same directory. The dataset is preprocessed using `dataset_preparation.py`, which you can modify if you wish to add custom preprocessing steps.
```

Comment:
```suggestion
You can download the TON_IoT dataset by accessing the following [link](https://research.unsw.edu.au/projects/toniot-datasets). Please navigate to `TON_IoT datasets/Train_Test_datasets/Train_Test_Linux_dataset` and download the file `Train_test_linux_memory.csv`. Then, rename the downloaded file to `dataset.csv` and place it in a new `dataset/` directory located in the directory where this `README.md` is. 

If you want to run the experiments with your own data, ensure your dataset is also named `dataset.csv` and located in the same directory. The dataset is preprocessed using `dataset_preparation.py`, which you can modify if you wish to add custom preprocessing steps.
```

Reply:
just indicating that the `dataset/` directory needs to be placed in the current dir.